lib/widgets/order_item.dart

```diff
@@ -0,0 +1,73 @@
+import 'dart:math';
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+
+import '../providers/orders.dart' as ord;
+
+class OrderItem extends StatefulWidget {
+  final ord.OrderItem order;
+
+  const OrderItem({@required this.order});
+
+  @override
+  _OrderItemState createState() => _OrderItemState();
+}
+
+class _OrderItemState extends State<OrderItem> {
+  var _expanded = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return Card(
+      margin: const EdgeInsets.all(10),
+      child: Column(
+        children: [
+          ListTile(
+            title: Text('\$${widget.order.amount}'),
+            subtitle: Text(
+              DateFormat('dd/MM/yyyy hh:mm').format(widget.order.dateTime),
+            ),
+            trailing: IconButton(
+              icon: Icon(_expanded ? Icons.expand_less : Icons.expand_more),
+              onPressed: () {
+                setState(
+                  () {
+                    _expanded = !_expanded;
+                  },
+                );
+              },
+            ),
+          ),
+          if (_expanded)
+            Container(
+              padding: const EdgeInsets.symmetric(horizontal: 15, vertical: 4),
```

Comment:
you might want to use EdgeInsets.all and set it to 16 instead, to fix the padding at the bottom.

Reply:
or better, please try to implement this widget having the expanding/collapse behavior to display what needs to be shown: https://api.flutter.dev/flutter/material/ExpansionTile-class.html

Reply:
sample reference: https://www.youtube.com/watch?v=saIoyINg64o